opm/simulators/flow/Main.hpp

```diff
@@ -80,6 +93,39 @@ NEW_TYPE_TAG(FlowEarlyBird, INHERITS_FROM(EclFlowProblem));
 
 END_PROPERTIES
 
+#ifndef OPM_FLOW_MAIN
```

Comment:
Shouldn't this compile regardless of the `#ifndef`? In that case could remove it here.

Reply:
Yes that code compiles regardless. But if we remove it, wouldn't the code then be included in the built `flow` executable which will never use the code? Adding unnecessary unused code and increasing the executable size (although very little) seems not good, right?

Reply:
As the method has a template parameter it will only be compiled if used.

Reply:
Of course, I did not think about that! Updated the PR with this commit https://github.com/OPM/opm-simulators/pull/2523/commits/125b0f11bc3f74d5e8650d268ad741662b3868c6